Lombiq.Tests.UI.Samples/UITestBase.cs

```diff
@@ -79,10 +80,17 @@ protected UITestBase(ITestOutputHelper testOutputHelper)
                 // disable it with the below config). With this, you can make sure that the HTML markup the app
                 // generates (also from content items) is valid. While the default settings for HTML validation are most
                 // possibly suitable for your projects, check out the HtmlValidationConfiguration class for what else
-                // you can configure. We've also added a .htmlvalidate.json file (note the Content Build
+                // you can configure. We've also added a custom .htmlvalidate.json file (note the Content Build
                 // Action) to further configure it.
                 ////configuration.HtmlValidationConfiguration.RunHtmlValidationAssertionOnAllPageChanges = false;
 
+                // This is to make sure that the HTML validation config file in this project takes effect. Only needed
+                // until we fix https://github.com/Lombiq/UI-Testing-Toolbox/issues/359.
+                configuration.HtmlValidationConfiguration.HtmlValidationOptions =
+                    configuration.HtmlValidationConfiguration.HtmlValidationOptions
+                        .CloneWith(validationOptions => validationOptions.ConfigPath =
+                            Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "samples.htmlvalidate.json")); // #spell-check-ignore-line
```

Comment:
This will need to be adapted if https://github.com/Lombiq/UI-Testing-Toolbox/pull/361 is merged first.